src/renderer/actions/addValidator.ts

```diff
@@ -0,0 +1,16 @@
+import {AddValidatorActionTypes} from "../constants/action-types";
+import {Action, Dispatch} from "redux";
+
+export const storeAddValidatorAction = (addValidator: boolean) =>
+    (dispatch: Dispatch<IAddValidatorAction>): void => {
+        dispatch(setAddValidator(addValidator));
+    };
+export const setAddValidator = (addValidator: boolean): IAddValidatorAction => ({
+    type: AddValidatorActionTypes.ADD_VALIDATOR, payload: {addValidator}
+});
+export interface IStoreAddValidatorPayload {
+    addValidator: boolean;
```

Comment:
is this just for navigation purposes in onboarding?

Might make more sense to use url params.